### PR TITLE
feat(command-feature): display an embed message error when duplicated flags are given

### LINKED ISSUE #903: [FEATURE] Feature command: error when duplicated flag
**Describe the solution you'd like**
Handling in parallel multiple times the same command is a pure no-go.
I think that if someone enter the same flag twice or via a shortcut we should tell him that he is doing wrong.
This is not the best friendly user action but throwing an error seems more appropriate anyway.

### LINKED ISSUE #925: [BUG] discordGetCommandFlags: --enabled=true -e --enabled=false fail
**Describe the bug**
When searching to split the flags,
This case case is failing: `--enabled=true -e --enabled=false`
Only the first two flags are returned.

**Expected behavior**
The three flags are returned.

### PR DESCRIPTION
**Pull request by C0ZEN**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](https://github.com/Sonia-corporation/il-est-midi-discord/blob/master/CONTRIBUTING.md)
- [x] Tests for the changes have been added/updated (for bugfix/feature)
- [ ] Docs have been added/updated (for bugfix/feature)

## PR Type
What kind of change does this PR introduce?

- [x] Feature (a new feature)
- [x] Bugfix (a bug fix)
- [ ] Style (changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc))
- [ ] Refactor (a code change that neither fixes a bug nor adds a feature)
- [ ] Perf (a code change that improves performance)
- [x] Test (adding missing tests or correcting existing tests)
- [ ] Build (changes that affect the build system, CI configuration or external dependencies)
- [ ] Docs (changes that affect the documentation)
- [x] Chore (anything else), please describe:

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

Closes #903.
Fixes #925.

### CODE REVIEW

File: src/features/discord/messages/classes/commands/flags/discord-command-flags.ts [hunk 1]
```diff
@@ -351,4 +389,103 @@ export class DiscordCommandFlags<T extends string> {
       name: DiscordCommandFlagErrorTitleEnum.UNKNOWN_FLAG,
     };
   }
+
+  private _getFlagsDuplicated(
+    messageFlags: Readonly<string>[]
+  ): IDiscordCommandFlagsDuplicated {
+    const active = this._getDuplicatedMessagesFlagsByName(messageFlags);
```
Comment (C0ZEN) [inline src/features/discord/messages/classes/commands/flags/discord-command-flags.ts:396]: Rename active and add type

File: src/features/discord/messages/services/command/feature/services/flags/discord-message-command-feature-duplicated-flags-error.service.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        }
+      ),
+      this._getMessageEmbedHintField()
+    );
+  }
+
+  private _getMessageEmbedHintField(): EmbedFieldData {
+    return {
+      name: `How to solve this?`,
+      value: `I am here to help you but do not mess with me!\nTry again but remove the extra duplicated flags and we can talk.`,
```
Comment (C0ZEN) [inline src/features/discord/messages/services/command/feature/services/flags/discord-message-command-feature-duplicated-flags-error.service.ts:106]:
```suggestion
      value: `I am here to help you but do not mess with me!\nTry again but remove the extra duplicated flags and then we can talk.`,
```

File: src/features/discord/messages/classes/commands/flags/discord-command-flags.ts (hunk 1 above)
Comment (C0ZEN) [inline src/features/discord/messages/classes/commands/flags/discord-command-flags.ts:396]:
```suggestion
    const messageFlagsWithName: Dictionary<IDiscordCommandMessageFlagWithName[]> = this._getDuplicatedMessagesFlagsByName(messageFlags);
```

**Review by C0ZEN — COMMENTED** [review]
Added two comments.

File: src/features/discord/messages/services/command/feature/services/flags/discord-message-command-feature-duplicated-flags-error.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+      > => {
+        expect.assertions(1);
+
+        const result = await service.getMessageResponse(flagsDuplicated);
+
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        expect(result.options.embed.fields[1]).toStrictEqual({
+          name: `How to solve this?`,
+          value: `I am here to help you but do not mess with me!\nTry again but remove the extra duplicated flags and we can talk.`,
```
Comment (C0ZEN) [inline src/features/discord/messages/services/command/feature/services/flags/discord-message-command-feature-duplicated-flags-error.service.spec.ts:221]:
```suggestion
          value: `I am here to help you but do not mess with me!\nTry again but remove the extra duplicated flags and then we can talk.`,
```

**Review by vvalembois — APPROVED** [review]

**Pull request merged**